tests/libunwind/libunwind.edl

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+enclave {
+    enum string_limit {
+        STRLEN_MAX = 200
+    };
+    
+    trusted {
+        public int test(
+            [out]char test[201],
+            uint32_t pid);
+    };
+    untrusted {
+        void exit_ocall(
+            uint64_t val);
```

Comment:
Can you just change the type to int? 
In my PR #1196  to compile with strict warning levels, I'm having to cast the val from uint64_t to int since the signature is exit(int).

https://github.com/Microsoft/openenclave/blob/c03338c072452cc9673d78f303e72691e4e4152e/tests/libunwind/host/host.cpp#L35-L38

Reply:
Of course.